components/aws/common/common.mjs

```diff
@@ -0,0 +1,124 @@
+import { EC2Client } from "@aws-sdk/client-ec2";
+import { LambdaClient } from "@aws-sdk/client-lambda";
+import { SSMClient } from "@aws-sdk/client-ssm";
+import { IAMClient } from "@aws-sdk/client-iam";
+import { S3Client } from "@aws-sdk/client-s3";
+import { CloudWatchLogsClient } from "@aws-sdk/client-cloudwatch-logs";
+import { EventBridgeClient } from "@aws-sdk/client-eventbridge";
+import { SQSClient } from "@aws-sdk/client-sqs";
+import { SNSClient } from "@aws-sdk/client-sns";
+import { SESClient } from "@aws-sdk/client-ses";
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+
+export default {
+  defaultRegion: "us-east-1",
+  awsClients: {
```

Comment:
By convention, files with the name `common.mjs` in this repo are used like base classes, with the same structure as components. (See the [guideline docs](https://pipedream.com/docs/components/guidelines/#common-files-optional))

I'd suggest renaming this file, or splitting this file into a `regions.mjs` file and `clients.mjs` file?

Reply:
Addressed